backend/app/api/v1/helper/helper.go

```diff
@@ -135,11 +135,6 @@ func CheckBind(req interface{}, c *gin.Context) error {
 	return nil
 }
 
-func ErrResponse(ctx *gin.Context, code int) {
-	ctx.JSON(code, nil)
-	ctx.Abort()
-}
-
 func ErrWithHtml(ctx *gin.Context, code int, scope string) {
 	if code == 444 {
 		ctx.String(444, "")
```

Comment:
I haven't found any errors with that current code. The function `ErrResponse` seems to be correct according to the provided implementation, and the variable naming is consistent across both files (though it's more common practice to use all uppercase for variables). No further optimizations suggested here since no issues were flagged and the context of optimization in this case might not exist unless we're referring specifically to database queries or CPU-bound logic tasks where optimization could apply.

Remember to always ensure that these changes don't break existing functionalities and compatibility with external dependencies.

Feel free to ask if there's something else you need help on!